tests/test_tree.py

```diff
@@ -87,6 +87,41 @@ def test_recursive_seed_part_returns_within_epsilon_of_target_pop(twelve_by_twel
                for part_pop in partition['pop'].values())
 
 
+def test_recursive_seed_part_uses_method(twelve_by_twelve_with_pop):
+    calls = 0
+
+    def dummy_method(graph, pop_col, pop_target, epsilon, node_repeats):
+        nonlocal calls
+        calls += 1
+        return bipartition_tree(
+            graph,
+            pop_col=pop_col,
+            pop_target=pop_target,
+            epsilon=epsilon,
+            node_repeats=node_repeats,
+            max_attempts=10000,
+        )
+
+    n_districts = 7  # 144/7 ≈ 20.5 nodes/subgraph (1 person/node)
+    ideal_pop = (
+        sum(
+            twelve_by_twelve_with_pop.nodes[node]["pop"]
+            for node in twelve_by_twelve_with_pop
+        )
+    ) / n_districts
+    epsilon = 0.1
+    result = recursive_seed_part(
+        twelve_by_twelve_with_pop,
+        range(n_districts),
+        ideal_pop,
+        "pop",
+        epsilon,
+        n=5,
+        ceil=None,
+        method=dummy_method,
+    )
+    assert calls == n_districts - 1
```

Comment:
This assertion is a little brittle and might break if the internal implementation of `recursive_seed_part` changes in the future. I think it's alright to have here, though.

Reply:
@InnovativeInventor Ty, great call out.

I updated it to be a `>=` check, since I think there's a stronger case to be made that `method` is called *at least* once is part of `recursive_seed_part`'s contract, let me know your thoughts.

Tbh I'd also be fine scrapping the unit test if you prefer, considering a regression here doesn't seem terribly likely.

Reply:
Great, LGTM.